README.md

```diff
@@ -19,10 +19,7 @@ TODO: Replace me with description of the module(s) in this repo
 <!-- Below content is automatically populated via pre-commit hook -->
 <!-- BEGIN OVERVIEW HOOK -->
 ## Overview
-* [terraform-ibm-module-template](#terraform-ibm-module-template)
-* [Examples](./examples)
-    * [Basic example](./examples/basic)
-    * [Complete example](./examples/complete)
+* [terraform-ibm-observability-da](#terraform-ibm-observability-da)
```

Comment:
Since there is no terraform module in the root directory, please update the README.md to match this one ->  https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/blob/main/README.md